public/controllers/management/components/management/ruleset/actions-buttons.js

```diff
@@ -119,7 +138,9 @@ class WzRulesetActionButtons extends Component {
       this.props.updateIsProcessing(true);
       this.props.updatePageIndex(0);
       this.props.updateLoadingStatus(false);
-    } catch (error) {};
+    } catch (error) {
+      // throw o no
```

Comment:
Yes, here we should catch this error and handler like `Error` level and `Business` severity and don't store.